src/main/java/com/acoustic/salarycalculator/printer/SalaryCalculatorPrinter.java

```diff
@@ -0,0 +1,60 @@
+package com.acoustic.salarycalculator.printer;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Arrays;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import com.acoustic.salarycalculator.calculator.SalaryCalculator;
+import com.acoustic.salarycalculator.jobscategories.JobsCategory;
+
+
+public class SalaryCalculatorPrinter {
+
+
+    public void printSalaryCalculatorTax(BigDecimal grossMonthlySalary) {
+        Arrays.stream(SalaryCalculator.values())
+                .forEach(values -> System.out.print(
+                        "\n" + values.getDescription() + values.getOperator().apply(grossMonthlySalary)));
+    }
+
+
+    public void printAverage(BigDecimal monthlyAverage, BigDecimal grossMonthlySalary) {
+        if (monthlyAverage != null) {
+            if (monthlyAverage.compareTo(grossMonthlySalary) > 0) {
+                System.out.println(
+                        "The monthly gross average is " + monthlyAverage.setScale(2, RoundingMode.HALF_EVEN) +
+                                " and your monthly gross is " + grossMonthlySalary.setScale(2, RoundingMode.HALF_EVEN) +
+                                " and it is below of the threshold");
+            } else if (monthlyAverage.compareTo(grossMonthlySalary) == 0) {
```

Comment:
DRY (Don't repeat yourself)
this is almost the same 3 times, there is just one word different, you could extract method with parameter Below, Equal , Above

Reply:
fixed